src/pages/List.js

```diff
@@ -7,7 +8,11 @@ export default function List({ token }) {
   const [listItem, loading, error] = useCollection(db.collection(token), {
     snapshotListenOptions: { includeMetadataChanges: true },
   });
+  const [filter, setFilter] = useState('');
```

Comment:
@apdsrocha @jssckbl I don't know why my comment about changing `filter` and `setFilter` is showing up twice. The first one I deleted, because I like `query` and `setQuery` better! But obviously whatever y'all decide works for me too!